### PR TITLE
fixing layout to reflect new desgn by @smombartz

### PR DESCRIPTION
**Pull request by julien51**
# Description

This design is sigificantly simpler to implement.
A follow up PR will add some cleanup for un-used components.
Also, we need to add the new tos prompt to the unlock-app for keychain and dashboard before merging this.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

image

File: unlock-protocol.com/src/components/content/HomeContent.js [hunk 1]
```diff
@@ -49,98 +53,120 @@ export const HomeContent = () => {
...
-        <Actions>
-          <h1>Monetize the Web with memberships</h1>
-          <Buttons>
-            <HomepageButton
-              label="Dashboard"
-              destination={`${
-                getConfig().publicRuntimeConfig.unlockApp
-              }/dashboard`}
+      <Grid>
+        <Hero>Monetize the web through Memberships</Hero>
```
Comment (cnasc) [inline unlock-protocol.com/src/components/content/HomeContent.js]: Should we make the casing more consistent on this? Either title case, or by downcasing “Memberships”

File: unlock-protocol.com/src/components/content/HomeContent.js [hunk 2]
```diff
@@ -49,98 +53,120 @@ export const HomeContent = () => {
...
+        <BannerKey
+          src="/static/images/illustrations/banner-key.png"
+          alt=""
+          width="192"
+        />
+        <SubTitle>
+          Unlock lets you let’s you easily lock and manage access to your
```
Comment (cnasc) [inline unlock-protocol.com/src/components/content/HomeContent.js]: Phoneposting so I can’t use the suggestion feature, but you have “lets you” twice on this line

File: unlock-protocol.com/src/components/content/HomeContent.js [hunk 3]
```diff
@@ -49,98 +53,120 @@ export const HomeContent = () => {
...
-            </HomepageButton>
-            <HomepageButton
-              label="Keychain"
-              destination={`${
-                getConfig().publicRuntimeConfig.unlockApp
-              }/keychain`}
+              Dashboard
+            </Button>
+            <p>
+              Create and manage locks to monetize access to your conentent,
```
Comment (cnasc) [inline unlock-protocol.com/src/components/content/HomeContent.js]: conentent -> content

File: unlock-protocol.com/src/components/content/HomeContent.js (hunk 2 above)
Comment (julien51) [reply unlock-protocol.com/src/components/content/HomeContent.js]: Dang. that's rigth! Thanks!

**Pull request merged**